src/drive/targets/mobile/README.md

```diff
@@ -123,47 +145,76 @@ On iOS:
 
 ## :lock: Create Release
 
-### Android
+### Get translations first
 
-Create these folders:
+Before building the application, you need to get the translations:
 
-```
-$ mkdir -p src/drive/targets/mobile/keys/android && mkdir -p src/drive/targets/mobile/build/android
+```sh
+$ yarn tx
 ```
 
-You must have this files in `src/drive/targets/mobile/keys/android`:
+### Android
 
-- cozy-drive-release-key.jks
-- key.json
+#### Requirements
 
-:warning: You must also have the password associated with `cozy-drive-release-key.jks`
+You must have `cozy-drive-release-key.jks` and `key.json` in `src/drive/targets/mobile/keys/android`, and the password associated with `cozy-drive-release-key.jks`
```

Comment:
can you say if those files are present in the password store? If yes, can you write the path to get them?